README.md

```diff
@@ -1,8 +1,9 @@
 Pa11y Runner
 ===
-`pa11y_runner` is a simple wrapper for running pa11y scans against local copies of websites.
+`pa11y_runner` is a simple wrapper for the [pa11y](https://pa11y.org/) CLI tool for running scans against html source files from websites dowloaded using the browser's "Save as" menu.
```

Comment:
```suggestion
`pa11y_runner` is a simple wrapper for the [pa11y](https://pa11y.org/) CLI tool for running scans against HTML source files from websites downloaded using the browsers "Save as" menu.
```

Reply:
Is there a tool that exists to download websites? Is the browsers "save as" the best way to do this?

Reply:
We used the browser because for this project, the website only works with older versions of IE.  All other browsers are not supported.  Even Legacy Edge doesn't work.  That's why we need to save the webpage source rather than just scanning the page directly.  I don't know if there is an IE library or similar that would work with Python, but if so, as soon as that's used, this application is no longer cross platform.